webpack.config.js

```diff
@@ -6,13 +6,13 @@ const APP_DIR = path.resolve(__dirname, "source/");
 const config =
 {
 	mode: "development",
-	target: "web",
+	target: "node",
```

Comment:
I wonder if we should make two different webpack config files. One for webpacking for node and one for web. Maybe webpack.node.config.js and webpack.web.config.js and then have two separate scripts in package.json for building each. Probably want to configure each to build a different .min.js file for each too.

Reply:
That's probably a good idea

Reply:
Although I wonder how other libraries handle this such that you can reference a single npm module for both. I think for now two versions of webpack.config is probably most efficient, but I would like to see if there's something inherently more efficient for this...